thrift/storypackage.thrift

```diff
@@ -0,0 +1,10 @@
+
+struct StoryPackageItem {
+  1: required string internalPageCode,
```

Comment:
I would remove them all. I have done so in the CAPI scala client: https://github.com/guardian/content-api-scala-client/blob/master/src/main/thrift/content/v1.thrift